corehq/apps/api/resources/v0_5.py

```diff
@@ -311,7 +316,10 @@ def obj_create(self, bundle, **kwargs):
                 created_via=USER_CHANGE_VIA_API,
                 email=bundle.data.get('email', '').lower(),
             )
-            del bundle.data['password']
+            # username cannot be updated
+            bundle.data.pop('username', None)
```

Comment:
will this ever get hit? if there is username in bundle.data, there will be an error already.

Reply:
This is in the `obj_create` method so there should be a `username` key. Definitely not my preferred way to handle this but was just making minimal changes for the time being.